app/helpers/metadata_helper.rb

```diff
@@ -169,9 +169,29 @@ def metadata_csv_has_duplicate_columns(columns, existing_fields, error_aggregato
     return has_duplicate_columns
   end
 
+  # Convenience wrapper
+  def validate_metadata_csv_for_project_samples(samples, metadata)
+    validate_metadata_csv_for_samples(samples, metadata, false, false)
+  end
+
+  # Convenience wrapper
+  def validate_metadata_csv_for_new_samples(samples, metadata)
+    # TODO: (gdingle): remove admin only after launch. See https://jira.czi.team/browse/IDSEQ-2051.
```

Comment:
add date?

Reply:
I think the link to the JIRA is sufficient.